src/service.rs

```diff
@@ -50,15 +49,15 @@ native_executor_instance!(
     pub AlcyoneExecutor,
     polymesh_runtime_testnet::api::dispatch,
     polymesh_runtime_testnet::native_version,
-    (frame_benchmarking::benchmarking::HostFunctions, native_rng::HostFunctions)
+    frame_benchmarking::benchmarking::HostFunctions,
```

Comment:
Don't we need `native_schnorrkel::HostFunctions` here?

Reply:
I've double-checked it after your question. 
`native_schnorrkell::HostFunctions` are used only in benchmarks (in develop network) to sign messages, so we don't need them in `Alcyone`.

Reply:
@maxsam4 I've tested a network upgrade from `develop`, and I've used `identity::mock_cdd_register_did` to verify that migration, and there is **not issues**.

Thinking twice, that is the expected result because:
 - The Tx is still valid because `hosted functions` are not part of the message serialization. :heavy_check_mark: 
 - `native_rng` is defined and used inside the removed `confidential` pallet. The generation of `CDDId` is completely isolated inside `confidential_identity` crate, and it does not use any `rng` as an input parameter. :heavy_check_mark: